### PR TITLE
[pt-br] update of `web/http/cookies`

### PR DESCRIPTION
**Pull request by vittxr**
- Improve `Lax` attribute description: I improved it based on this Stack Overflow [answer](https://stackoverflow.com/a/59995877/18432809).
- Fix `note` card with the backend languages: it is equal to [English](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies) docs.
- Fix `SameSite` description (in the [English](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies) docs, `Lax` is the default value of `SameSite`).

### CODE REVIEW

File: files/pt-br/web/http/cookies/index.md [hunk 1]
```diff
@@ -34,11 +34,7 @@ Set-Cookie: <cookie-name>=<cookie-value>
 
 Este cabeçalho de servidor informa ao cliente para armazenar um cookie.
 
-> **Nota:** Eis as formas de utilização do cabeçalho `Set-Cookie` em várias aplicações de servidor:- [PHP](https://secure.php.net/manual/en/function.setcookie.php)
->
-> - [Node.JS](https://nodejs.org/dist/latest-v8.x/docs/api/http.html#http_response_setheader_name_value)
-> - [Python](https://docs.python.org/3/library/http.cookies.html)
-> - [Ruby on Rails](http://api.rubyonrails.org/classes/ActionDispatch/Cookies.html)
+> **Nota:** Eis as formas de utilização do cabeçalho `Set-Cookie` em várias aplicações de servidor: [PHP](https://secure.php.net/manual/en/function.setcookie.php), [Node.JS](https://nodejs.org/dist/latest-v8.x/docs/api/http.html#http_response_setheader_name_value), [Python](https://docs.python.org/3/library/http.cookies.html), [Ruby on Rails](http://api.rubyonrails.org/classes/ActionDispatch/Cookies.html)
```
Comment (josielrocha) [inline files/pt-br/web/http/cookies/index.md:37]:
```suggestion
> [!NOTE]
> Eis as formas de utilização do cabeçalho `Set-Cookie` em várias aplicações de servidor: [PHP](https://secure.php.net/manual/en/function.setcookie.php), [Node.JS](https://nodejs.org/dist/latest-v8.x/docs/api/http.html#http_response_setheader_name_value), [Python](https://docs.python.org/3/library/http.cookies.html), [Ruby on Rails](http://api.rubyonrails.org/classes/ActionDispatch/Cookies.html)
```

**Review by josielrocha — COMMENTED** [review]
Hey @vittxr!

Thank you for your help!

We are changing the way we write note blocks. Can you please check the suggestion I left in this PR?

**Comment by vittxr** [comment]
@josielrocha No problem! I'm glad to help :) 

I approved the suggestions

**Comment by josielrocha** [comment]
Thanks a lot for your help @vittxr!

**Pull request merged**